python/lsst/cell_coadds/_image_planes.py

```diff
@@ -116,6 +116,22 @@ def __init__(
         self._mask_fractions = mask_fractions
         self._noise_realizations = tuple(noise_realizations)
 
+    @classmethod
+    def from_exposure(
+        cls,
+        exposure: Exposure,
```

Comment:
This doesn't actually use anything from the `Exposure` that isn't part of `MaskedImage`.  Better to call this `from_masked_image` and just take `MaskedImage`.

Reply:
I do want to reduce usage of `Exposure` when it's not adding _much_ on top of `MaskedImage`, because I'd like to someday diminish its use to the point where we could drop it.

But even if we had no intention of dropping it ever, it'd still be inappropriate to use it in places where it's adding _nothing_ to  `MaskedImage`, as is the case here and in the `drp_tasks` branch; this is the [Interface Segregation Principle](https://en.wikipedia.org/wiki/Interface_segregation_principle).  It's the same reason `AccumulatorMeanStack` only deals with `MaskedImage`, not `Exposure`.

(Yes, the `drp_tasks` branch does call `getWcs`, but it also has that same WCS throughout the exposure's lifetime already in the local `skyInfo` variable, so it doesn't have to.)